code/modules/mob/hear_say.dm

```diff
@@ -154,16 +154,16 @@
 		return
 
 	if(say_understands(speaker, language))
-		message = "<B>[src]</B> [verb], \"[message]\""
+		message = "<span class='game say'><span class='name'>[src]</span> [verb], <span class='message'><span class='body'>\"[message]\"</span></span></span>"
 	else
-		message = "<B>[src]</B> [verb]."
+		message = "<span class='game say'><span class='name'>[src]</span> [verb] with their hands, but you don't understand.</span>"
```

Comment:
Span class 'name'?

Reply:
https://github.com/ParadiseSS13/Paradise/blob/a4d536bdaa11cc70b77bbce99289bd005c568001/goon/browserassets/css/browserOutput.css#L215

the .name class just makes it bold, and is consistent with the way the output of the hear_say proc is formatted.